.gitignore

```diff
@@ -55,6 +55,7 @@ util/Win_Check_Output.txt
 .vscode/last.sql
 .vscode/temp.sql
 .vscode/ipch/
+.vscfavoriterc
```

Comment:
Could this be removed/reverted? 

If it's something that should be added, it should be done so in a separate PR from the keymaps

Reply:
sure, I can just move it to my `~/.config/git/ignore` for now